uninit_filter.py

```diff
@@ -26,10 +29,25 @@ def find_uninit_classes(coverage_file):
     return uninit_dict
 
 
+def output(uninit_dict, output_file):
+    if output_file is not None:
+        with open(output_file, 'w') as out:
+            for file in uninit_dict:
+                out.write(file + str(uninit_dict[file]) + "\n")
+    else:
+        for file in uninit_dict:
```

Comment:
Since you are referencing a value in the dict, ie `dict[key]` in the loop block I think it would be better to use `dict.iteritems()` or `dict.items()`.  This allows you to unpack the variable names in the loop definition line.  Worth noting that in python3 `.items()` makes a copy of the dict so I think `iteritems()` is more performant.  Maybe check me on that?

http://stackoverflow.com/questions/10458437/what-is-the-difference-between-dict-items-and-dict-iteritems

The line would become `for f_name, cls_name in uninit_dict.iteritems():`